features/set-as-default-and-add-to-dock.json

```diff
@@ -0,0 +1,13 @@
+{
+    "_meta": {
+        "description": "Control for SAD and ATT prompts on macOS",
+        "sampleExcludeRecords": {}
+    },
+    "state": "enabled",
+    "exceptions": [],
+    "features": {
+        "popoverVsBannerExperiment": {
+            "state": "enabled"
+        }
+    }
```

Comment:
@jotaemepereira it’s a good practice to disable flags by default and only enable them in the overrides file. Otherwise, as you can see in the auto-generated PR comment, other platforms got this feature flag enabled.

Reply:
Done. Thanks for the feedback 🙇🏼‍♂️